packages/workspace/src/browser/workspace-service.ts

```diff
@@ -95,8 +95,8 @@ export class WorkspaceService implements FrontendApplicationContribution {
                 this.updateWorkspace();
             }
         });
-        this.fsPreferences.onPreferenceChanged(e => {
-            if (e.preferenceName === 'files.watcherExclude') {
+        this.fsPreferences.onPreferenceChanged(event => {
+            if (event.preferenceName === 'files.watcherExclude') {
```

Comment:
I believe the change is unnecessary, in other places (earlier in the pr we have `e`). If we were to make the update we should probably follow one format.

Reply:
Just aligned with another listener from the same file, I did not touch other places for the reason you mentioned.